website/docs/r/region.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_region"
+sidebar_current: "docs-vcfa-resource-region"
+description: |-
+  Provides a resource to manage Regions in VMware Cloud Foundation Automation.
+---
+
+# vcfa\_region
+
+Provides a resource to manage Regions in VMware Cloud Foundation Automation.
+
+~> Only `System Administrator` can create this resource.
+
+## Example Usage
+
+```hcl
+data "vcfa_vcenter" "one" {
+  name = "vcenter-one"
+}
+
+data "vcfa_supervisor" "one" {
+  name       = "my-supervisor-name"
+  vcenter_id = data.vcfa_vcenter.one.id
+}
+
+resource "vcfa_region" "one" {
+  name                 = "region-one"
+  nsx_manager_id       = data.vcfa_nsx_manager.test.id
```

Comment:
Thinking we should have it in the example, like we have supervisor data source, too.